### PR TITLE
13.0 mig account invoice import

### PR DESCRIPTION
**Pull request by etobella**

### CODE REVIEW

File: account_invoice_import/models/account_invoice_import_config.py [hunk 1]
```diff
@@ -37,9 +39,7 @@ class AccountInvoiceImportConfig(models.Model):
         "res.company",
         ondelete="cascade",
         required=True,
-        default=lambda self: self.env["res.company"]._company_default_get(
-            "account.invoice.import.config"
-        ),
+        default=lambda self: self.env.company,
```
Comment (etobella) [inline account_invoice_import/models/account_invoice_import_config.py:42]: Fixing a raised warning

File: account_invoice_import/models/account_invoice_import_config.py [hunk 2]
```diff
@@ -91,7 +97,7 @@ def invoice_line_method_change(self):
 
     @api.onchange("partner_id")
     def partner_id_change(self):
-        # if available get proporty
+        # if available get property
```
Comment (etobella) [inline account_invoice_import/models/account_invoice_import_config.py:100]: Typo error

File: base_business_document_import/components/edi_input_process_business_document.py [hunk 3]
```diff
@@ -0,0 +1,163 @@
+# Copyright 2020 Creu Blanca
+# @author: Enric Tobella
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.addons.component.core import AbstractComponent
+
+
+class EDIInputProcessBusinessDocumentImport(AbstractComponent):
```
Comment (etobella) [inline base_business_document_import/components/edi_input_process_business_document.py:8]: This functions are taken from `business.document.import`. On future, they should need to be passed here :wink:

**Comment by etobella** [comment]
By the way, we could add this hooks
https://github.com/OCA/edi/pull/205

**Comment by rafaelbn** [comment]
@NL66278 Ronald, hello! what do you think about this one?

**Pull request closed without merging**